css/css.go

```diff
@@ -1201,6 +1201,41 @@ func (c *cssMinifier) minifyProperty(prop Hash, values []Token) []Token {
 			values[0].Data = oneBytes
 			values[0].Ident = 0
 		}
+	case UnicodeRange:
+		// U+2600-26FF → U+26??
+		for i, value := range values {
+			// Starts with "U+..." or "u+..."
+			if len(value.Data) >= 2 && value.Data[0]|32 == 'u' && value.Data[1] == '+' {
+				hyphen := bytes.IndexByte(value.Data, '-')
```

Comment:
Try to avoid function in `bytes` like these, as this makes it easier to write code non-optimally. For example, in this case you should parse from left to right and put the bytes in `left`. Then if you encounter `-`, stop parsing and put the remainder in `right`.